Functions/QualityControl/xASL_qc_GenerateReport.m

```diff
@@ -0,0 +1,418 @@
+function [x,config] = xASL_qc_GenerateReport(x, subject)
+% xASL_qc_GenerateReport Print output QC
+%
+% FORMAT: xASL_qc_GenerateReport(x)
+%
+% INPUT:
+%   x           - structure containing fields with all information required to run this submodule (REQUIRED)
+%   subject     - subject name (OPTIONAL, default = x.SUBJECT)
+%
+% OUTPUT: n/a
```

Comment:
function definition on the first line contains outputs.

Reply:
Added